tests/trimming_input/assert_show_output.py

```diff
@@ -0,0 +1,64 @@
+"""
+Module holding the correct values for show CLI command outputs for the trimming_test.py
+"""
+
+show_trim_empty="""\
+No configuration is present in CONFIG DB
+"""
+
+show_trim_partial="""\
++-----------------------------+---------+
+| Configuration               | Value   |
++=============================+=========+
+| Packet trimming size        | 200     |
++-----------------------------+---------+
+| Packet trimming DSCP value  | 20      |
++-----------------------------+---------+
+| Packet trimming queue index | N/A     |
```

Comment:
Do we allow this partial config??

Reply:
@kperumalbfn yes. If some parts are omitted, vendor SAI defaults are used